src/System.Text.Json/tests/Serialization/PolymorphicTests.cs

```diff
@@ -2,14 +2,113 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Json.Serialization.Tests
 {
     public static class PolymorphicTests
     {
         [Fact]
-        public static void Standard()
+        public static void PrimitivesAsRootObject()
+        {
+            string json = JsonSerializer.ToString<object>(1);
+            Assert.Equal("1", json);
+
+            json = JsonSerializer.ToString(1, typeof(object));
+            Assert.Equal("1", json);
+
+            json = JsonSerializer.ToString<object>("foo");
+            Assert.Equal(@"""foo""", json);
+
+            json = JsonSerializer.ToString("foo", typeof(object));
+            Assert.Equal(@"""foo""", json);
+        }
```

Comment:
Are `true`, `false`, `null`, or `3.141592653589793238462643383279m` (Pi as a decimal, to more precision that double can roundtrip) interesting here?

Reply:
Sure I added some of your examples. Thanks. However `pi` must be `3.1415926535897932384626433833m` irregardless of serialization (verified by just assigning it to a decimal and inspecting).